fixtures/shehds/led-flat-par-7x18w-rgbwa-uv-light.json

```diff
@@ -0,0 +1,164 @@
+    ],
+    "manual": [
+      "https://www.shehds.com/u_file/2005/file/b01e72a1b3.pdf"
+    ]
+  },
+  "physical": {
+    "dimensions": [176, 178, 11.3],
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "power": 126,
+    "weight": 0.9
+  },
+  "availableChannels": {
+    "Master dimmer": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Amber": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "Violet": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Strobe": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Strobe",
+        "speedStart": "0Hz",
+        "speedEnd": "10Hz",
+        "helpWanted": "At which DMX values is the strobe effect disabled?"
+      }
+    },
+    "Function": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 50],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [51, 100],
+          "type": "Effect",
+          "effectName": "Auto Mode 1",
+          "helpWanted": "At which DMX value does the effect start?"
+        },
+        {
+          "dmxRange": [101, 150],
+          "type": "Effect",
+          "effectName": "Auto Mode 2",
+          "helpWanted": "At which DMX value does the effect start?"
+        },
+        {
+          "dmxRange": [151, 200],
+          "type": "Effect",
+          "effectName": "Auto Mode 3",
+          "helpWanted": "At which DMX value does the effect start?"
+        },
+        {
+          "dmxRange": [201, 255],
+          "type": "Effect",
+          "soundControlled": true,
+          "effectName": "Sound Mode",
+          "helpWanted": "At which DMX value does the effect start?"
+        }
+      ]
+    },
+    "Function Speed": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Speed",
+        "speedStart": "slow",
+        "speedEnd": "fast"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "6-channel",
+      "shortName": "6ch",
+      "channels": [
+        "Red",
+        "Green",
+        "Blue",
+        "White",
+        "Amber",
+        "Violet"
+      ]
+    },
+    {
+      "name": "10-channel",
```

Comment:
In #1831 (which I closed as duplicate of this PR), there is only a 9-channel mode. I thought at the time that the 10-channel mode was just not entered completely. But there is also a (different) linked manual that only lists that 9-channel mode, while the fixture name is the same and the images look identical.

To resolve this, please merge the two fixture definitions into this PR:

* Include that 9-channel mode in this fixture.
* Add the manual from [#1831](https://github.com/OpenLightingProject/open-fixture-library/pull/1831) to the `manuals` section.
* Add René Bütikofer to the fixture authors.
* Add a fixture `comment` to explain the whole mess.
* Maybe something else that I forgot?

Reply:
I've applied these recommendations.